util/test/paratest.chapcs

```diff
@@ -78,13 +78,15 @@ def get_exclusive_nodes():
 def get_num_non_exclusive_nodes():
     """
     Get the number of nodes available for testing on the chapel partition
-    (total - exclusive)
+    (online - exclusive)
     """
+    online_states = 'ALLOC,ALLOCATED,COMP,COMPLETING,IDLE,MIX,MIXED'
```

Comment:
I think the variable name `online_node_states` would better describe this string